server/integration_tests/test_data/detection_api_context/stateswithhighestphds/chart_config.json

```diff
@@ -1,6 +1,16 @@
 {
   "childEntityType": "State",
   "classifications": [
+    {
```

Comment:
I'm not really sure what this section of this output means. @beets Any pointers?